### ISSUE TITLE
[Accessibility] Ctrl+M (editor.action.toggleTabFocusMode) does not work inside terminal input field

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#172024]
CC @meganrogge 

## Reproducible Steps

1. Load screen reader.

1. Open terminal.

1. Focus in terminal input area.

1. Press Ctrl+M to toggle tab-key move (`editor.action.toggleTabFocusMode`).

* Notes: I don't hear any change. It does not seem to work.

## Expected Behavior

Pressing Ctrl+M key needs to toggle the tab-key move mode.

## System Info

Version: 1.75.0-insider (system setup)
Commit: f1b09237cd400d92987cf7a9681995e8011ef54c
Date: 2023-01-23T11:53:59.088Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by meganrogge** [comment]
This works for me when I add `editor.action.toggleTabFocusMode` to `terminal.integrated.commandsToSkipShell`, so we can do that

**Comment by meganrogge** [comment]
On macOS, this works for me. It's part of our default `terminal.integrated.commandsToSkipShell` so surprising that it wouldn't work. will try windows

**Comment by meganrogge** [comment]
Yep does not work on windows

**Comment by meganrogge** [comment]
the logs show that the keybinding works and ` Invoking command editor.action.toggleTabFocusMode.` but then it doesn't happen

```
2023-01-25 06:25:58.152 [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 17, key: Control
2023-01-25 06:25:58.152 [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 5 ('Ctrl')
2023-01-25 06:25:58.153 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-25 06:25:58.301 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-25 06:25:58.302 [info] [KeybindingService]: | Resolving ctrl+M
2023-01-25 06:25:58.304 [info] [KeybindingService]: \ From 1 keybinding entries, matched editor.action.toggleTabFocusMode, when: no when condition, source: built-in.
2023-01-25 06:25:58.306 [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: KeyM, keyCode: 77, key: m
2023-01-25 06:25:58.306 [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: KeyM, keyCode: 43 ('M')
2023-01-25 06:25:58.307 [info] [KeybindingService]: | Resolving ctrl+M
2023-01-25 06:25:58.307 [info] [KeybindingService]: \ From 1 keybinding entries, matched editor.action.toggleTabFocusMode, when: no when condition, source: built-in.
2023-01-25 06:25:58.308 [info] [KeybindingService]: + Invoking command editor.action.toggleTabFocusMode.
2023-01-25 06:25:58.447 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-25 06:25:58.448 [info] [KeybindingService]: | Resolving ctrl+M
2023-01-25 06:25:58.449 [info] [KeybindingService]: \ From 1 keybinding entries, matched editor.action.toggleTabFocusMode, when: no when condition, source: built-in.
```

**Comment by meganrogge** [comment]
it only gets run when there's an editor view model, which there isn't when the terminal is focused

https://github.com/microsoft/vscode/blob/fc5d2e8d7ef21179bf86c79a7755b426b8a3baea/src/vs/editor/browser/coreCommands.ts#L34-L46
